sway-core/src/control_flow_analysis/dead_code_analysis.rs

```diff
@@ -414,7 +414,9 @@ fn connect_declaration(
             connect_storage_declaration(storage, graph, entry_node, tree_type);
             Ok(leaves.to_vec())
         }
-        ErrorRecovery | GenericTypeForFunctionScope { .. } => Ok(leaves.to_vec()),
+        ErrorRecovery | GenericTypeForFunctionScope { .. } | Break | Continue => {
```

Comment:
Here, we should return `vec![]` I think, since break and continue both abort the flow.

Reply:
Makes sense. Updated!